sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -1,75 +1,99 @@
 <template>
-  <v-row class="m-2">
-    <v-col class="category-header-medium color-blue">{{ header }}</v-col>
-  </v-row>
-  <div class="mx-5 py-4">
-    <content-group>
-      <DataTable
-        :autoLayout="true"
-        :value="applications"
-        class="p-m-4"
-        :paginator="true"
-        :rows="10"
+  <body-header
```

Comment:
There is no page header for this page?

Reply:
No this page is a child of  LocationCOESummary.vue which has page header.